### DISCUSSION TITLE
Translation of documentation into Spanish

### DISCUSSION THREAD

**Discussion by CristianjrmDEV** [sequelize/sequelize#16026] [Help & Questions]
Hello, I was wondering if you already had plans in motion to translate the documentation into Spanish. Otherwise, I would be happy to start the translation progress to bring Sequelize to the Spanish speaking community.

**Comment by WikiRik** [comment]
We do not have any plans for (un)official translations into other languages. We would recommend waiting a bit until the docs for v7 has gone through the improvements we're working on right now. See mainly https://github.com/sequelize/website/pull/330
